packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -166,31 +171,77 @@ class DescopeWc extends BaseDescopeWc {
       }
     | undefined;
 
-  async loadSdkScripts() {
-    const flowConfig = await this.getFlowConfig();
-    const scripts = flowConfig.sdkScripts;
-
-    scripts?.forEach(async (script) => {
-      const module = await loadSdkScript(script.id);
-      module(
-        script.initArgs as any,
-        { baseUrl: this.baseUrl },
-        (result: string) => {
-          // update the context with the result, under the `resultKey` key
-          this.dispatchEvent(
-            new CustomEvent('components-context', {
-              detail: {
-                // we store the result with script.id prefix to avoid conflicts with other scripts results
-                // that may have the same key
-                [getScriptResultPath(script.id, script.resultKey)]: result,
-              },
-              bubbles: true,
-              composed: true,
-            }),
-          );
+  #scriptsRegistry = new Map<string, ScriptModule>();
```

Comment:
Don't you prefer to handle by adding id to the script tag? 
if there are 2 flows instances on the same page we might get an error
I worked hard in order to support it 😄

Reply:
Fixed